spec/views/carts/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "carts/edit.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
ça ne bloque plus maintenant, les pending tests?

Reply:
It never did